plugins/woocommerce/includes/wc-core-functions.php

```diff
@@ -1048,6 +1048,10 @@ function wc_print_js() {
  * @param  bool    $httponly Whether the cookie is only accessible over HTTP, not scripting languages like JavaScript. @since 3.6.0.
  */
 function wc_setcookie( $name, $value, $expire = 0, $secure = false, $httponly = false ) {
+	if ( apply_filters( 'woocommerce_cookie_consent', true, $name ) === false && !Constants::is_true( 'WP_DEBUG' ) ) {
```

Comment:
```suggestion
	if ( ! apply_filters( 'woocommerce_set_cookie_enabled', true, $name ,$value, $expire, $secure ) ) {
```

I think we should rename this filter since it's a pretty generic cookie escape condition, cookie consent notification is just one of the use case. Also suggestion to add all the params instead of just `$name`.

Reply:
Looks good to me. Do you put the changes or should I submit them for the PR?

Reply:
I can commit them from the GH interface, no worries.